app/models/services/crossref/resource.rb

```diff
@@ -0,0 +1,52 @@
+class Crossref
+  class Resource
+    attr_accessor :id, :response
+
+    def initialize id
+      self.id = id.to_s
+      endpoint = URI.parse("http://api.crossref.org/works/#{self.id}")
```

Comment:
I was confused because it seemed redundant to do id.to_s and the use the string interpolation, but I assume you're using ID somewhere else.